assets/src/components/cd/services/service/ServiceErrors.tsx

```diff
@@ -38,14 +40,22 @@ export const ColMessage = columnHelper.accessor((row) => row?.message, {
   header: 'Message',
   enableSorting: true,
   meta: { gridTemplate: '6fr' },
-  cell: ({ getValue }) => (
-    <Code
-      maxHeight="300px"
-      width="100%"
-    >
-      {getValue()}
-    </Code>
-  ),
+  cell: function Cell({ getValue }) {
```

Comment:
how big is the encapsulating modal?  We should definitely expand the width since these messages do get long

Reply:
fixed, will expand if needed while staying within viewport